src/util.test.ts

```diff
@@ -33,27 +34,41 @@ describe("Util", (): void => {
       ): any => {
         callback(error, { stdout, stderr });
       }));
-      const output = await util.execBashCommand(command);
+
+      const output = await util.execBashCommand(command, platform);
 
       expect(core.info).nthCalledWith<[string]>(1, command);
+      const shell =
+        platform === "win32"
+          ? "C:\\Program Files\\Git\\bin\\bash.exe"
+          : "/usr/bin/bash";
       expect(child_process.exec).lastCalledWith(
         command,
-        { shell: "/usr/bin/bash" },
+        { shell },
         expect.anything()
       );
       return output;
     };
 
+    const platform = () => constantFrom<NodeJS.Platform>("linux", "win32");
```

Comment:
Please specify the return type since ESLint will likely complain about that once it awakens from its slumber (c.f., oxsecurity/megalinter#1572).